tests/test_search_params.py

```diff
@@ -81,6 +99,18 @@ def do_test_id_selector(self, index_key, id_selector_type="batch", mt=faiss.METR
                 if i not in ssubset
             ]).astype('int64')
             sel = faiss.IDSelectorNot(faiss.IDSelectorBatch(inverse_subset))
+        elif id_selector_type == "or":
+            lhs_sel = faiss.IDSelectorBatch(lhs_subset)
+            rhs_sel = faiss.IDSelectorBatch(rhs_subset)
+            sel = faiss.IDSelectorOr(lhs_sel, rhs_sel)
```

Comment:
to check if the python references worked correctly, please use 

```
sel = faiss.IDSelectorOr(
    faiss.IDSelectorBatch(lhs_subset), 
    faiss.IDSelectorBatch(rhs_subset)
)
```

This will crash if the references are not tracked correctly

Reply:
Not obvious the distinction between the two as my python is pretty basic, could you explain or provide a pointer to the difference between the two?

Reply:
I added a bit of doc about this in the wiki 

https://github.com/facebookresearch/faiss/wiki/Faiss-code-structure#object-ownership-python

Reply:
Thanks for the link! Makes more sense now.